var/spack/repos/builtin/packages/suntans/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+class Suntans(MakefilePackage):
+    """The Stanford unstructured-grid, 
+    nonhydrostatic, parallel coastal 
+    ocean model. """
+
+    homepage = "https://github.com/ofringer/suntans"
+    url      = "https://github.com/ofringer/suntans/archive/master.zip"
+
+    maintainers = ['ofringer', 'zyaj', 'mrayson', 'jadelson', 'lhxone']
+
+    version('0.0.1', '9b5d0dee5502a5650e37bf72f64b1124be86c084e7408aa517c214ddb25e36de')
+
+    depends_on('libx11@1.6.3', type = ('build', 'run'))
+    depends_on('openmpi', type = ('build'))
```

Comment:
Why the dependency is on OpenMPI specifically? It usually make more sense to:
```python
depends_on('mpi')
```
Not that a `type='build'` dependency is very unlikely, since usually program link to the underlying MPI library.

Reply:
solved